SPECS/kubernetes/kubernetes-1.17.13.spec

```diff
@@ -18,13 +18,17 @@ URL:            https://mcr.microsoft.com/oss
 #               Note that only amd64 tarball exist which is OK since kubernetes is built from source
 Source0:        kubernetes-node-linux-amd64-%{version}.tar.gz
 Source1:        kubelet.service
+Source2:        golang-1.15-k8s-1.17-test.patch
```

Comment:
Same comment as above - shouldn't this be a Patch# instead of Source#?

Reply:
patch cannot be used because of kubernetes tarball structure (src are in a tarball within the src tarball)